docker/entrypoints/proxito.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+
+../../docker/common.sh
+
+python3 manage.py runserver 0.0.0.0:8000
```

Comment:
Shouldn't proxito run on a different port than webs?

Reply:
There is no need for that since `proxito` is a different container than `web`, so there is no conflict there. One will be `http://proxito:8000/` and the other `http://web:8000/`.

If that's not what you are asking, I'm missing something.

Reply:
I just find it a bit simpler to think about when they're on diff ports, but not an issue for now.